### PR TITLE
add cron jobs in scheduler quickstart

### PR DESCRIPTION
**Pull request by machi1990**
Following https://github.com/quarkusio/quarkus/pull/5127
/cc @gsmet

### CODE REVIEW

File: scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java [hunk 1]
```diff
@@ -20,4 +21,13 @@ void increment() {
         counter.incrementAndGet();
     }
     
-}
\ No newline at end of file
+    @Scheduled(cron="0 15 10 * * ?")
```
Comment (cescoffier) [inline scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java]: Would it be possible to make it testable and extend the tests?

File: scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java [hunk 2]
```diff
@@ -20,4 +21,13 @@ void increment() {
...
+    void cronJob(ScheduledExecution execution) {
+        System.out.println(execution.getScheduledFireTime());
+    }
+
+    @Scheduled(cron = "{cron.expr}")
+    void cronJobWithExpressionInMpConfig() {
```
Comment (cescoffier) [inline scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java]: InConfig, instead of inMpConfig

**Comment by machi1990** [comment]
> Was the guide also updated?

Yes, the guide has been updated here https://github.com/quarkusio/quarkus/pull/5127

File: scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java (hunk 1 above)
Comment (machi1990) [reply scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java]: I am thinking about incrementing the counter for each of the two added methods. Is that acceptable?

Comment (cescoffier) [reply scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java]: +1

Comment (machi1990) [reply scheduler-quickstart/src/main/java/org/acme/scheduling/CounterBean.java]: Done, I have updated the two methods with a call that increment the counter. 
Same modification applied in the guide.

**Comment by gsmet** [comment]
Force pushed to fix a minor indentation issue and merged.

**Pull request merged**